### ISSUE TITLE
Great Works not showing properly and cannot move

### ISSUE THREAD

**Issue by cairnsy44** [LoneGazebo/Community-Patch-DLL#1687]
Bug Report Template

1. Mod Version (i.e Date - (4/23b)): 4/23 b

2. Mod List (if using standard CPP set, leave blank):
Standard CPP
JFD's Roosevelt
Sukritact's E/D v 3

3. Type of error (i.e. crash, interface bug, AI quirk): 
When I go to look at my great works by clicking on the tourism icon, Chicago pops up with some great works of art which I cannot look at or move. Washington is blank, even though I have some buildings in NY that should have great writings and a great music work.
Basically, the great works screen is not showing properly.

4. Steps to reproduce:

5. Additional information:

---------------------------

Supporting information:

Please note that you can attach .zip files to Issues by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files: Database.log and Lua.log needed
2. Minidump file (located in your Civ5.exe directory)
3. Screenshots (if needed)

----------------------------

**Comment by LoneGazebo** [comment]
Sounds like a mod conflict. If you have logging enabled, please load the game, attempt to check the art, and then post your lua.log here.

**Comment by JFDaly** [comment]
@cairnsy44 Which Roosevelt? Teddy, or Franklin?

**Comment by cairnsy44** [comment]
@JFDaly     FDR
 @LoneGazebo  ok...I will give it a shot next time I play.
